python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -50,6 +71,26 @@ def aggregate(self, arg, split_every=None, split_out=1):
         )
 
 
+class CudfSeriesGroupBy(SeriesGroupBy):
+    def __init__(self, *args, **kwargs):
+        self.sep = kwargs.pop("sep", "___")
+        self.as_index = kwargs.pop("as_index", True)
+        super().__init__(*args, **kwargs)
+
+    def mean(self, split_every=None, split_out=1):
+        return groupby_agg(
+            self.obj,
+            self.index,
+            {self._slice: "mean"},
+            split_every=split_every,
+            split_out=split_out,
+            dropna=self.dropna,
+            sep=self.sep,
+            sort=self.sort,
+            as_index=self.as_index,
+        )[self._slice]
```

Comment:
Can this be replicated on the `CudfDataFrameGroupBy` class?

Reply:
Yeah - For `CudfDataFrameGroupBy.mean`, we can just pass something like `{c: "mean" for c in self.obj.columns if c not in self.index}`